subsys/net/openthread/Kconfig.defconfig

```diff
@@ -260,6 +264,46 @@ endif # OPENTHREAD_CSL_RECEIVER
 
 endif # !OPENTHREAD_THREAD_VERSION_1_1
 
+if OPENTHREAD_COPROCESSOR_RCP
+
+config MBEDTLS_PSA_KEY_SLOT_COUNT
+	int
+	default 1
+
+config OPENTHREAD_COPROCESSOR_UART_RING_BUFFER_SIZE
+	int
+	default 2048
+
+config ISR_STACK_SIZE
+	int
+	default 640
+
+config IDLE_STACK_SIZE
+	int
+	default 128
+
+config MPSL_WORK_STACK_SIZE
+	int
+	default 512
+
+config OPENTHREAD_THREAD_STACK_SIZE
```

Comment:
What about moving this one and `OPENTHREAD_RADIO_WORKQUEUE_STACK_SIZE` to the `Kconfig` file next to this one? Not to have these configs scattered across different files.

Maybe `OPENTHREAD_COPROCESSOR_UART_RING_BUFFER_SIZE` as well.